tests/integration/test_backups.py

```diff
@@ -99,277 +100,110 @@ async def cloud_configs(ops_test: OpsTest, github_secrets) -> None:
+    # Ensure stanza is working correctly.
+    logger.info("listing the available backups")
+    action = await ops_test.model.units.get(new_unit_name).run_action("list-backups")
+    await action.wait()
+    backups = action.results.get("backups")
+    assert backups, "backups not outputted"
+    await ops_test.model.wait_for_idle(status="active", timeout=1000)
 
-            # Scale up to be able to test primary and leader being different.
-            async with ops_test.fast_forward():
-                await scale_application(ops_test, database_app_name, 2)
-
-            # Ensure replication is working correctly.
-            new_unit_name = f"{database_app_name}/2"
-            address = get_unit_address(ops_test, new_unit_name)
-            with db_connect(
-                host=address, password=password
-            ) as connection, connection.cursor() as cursor:
-                cursor.execute(
-                    "SELECT EXISTS (SELECT FROM information_schema.tables"
-                    " WHERE table_schema = 'public' AND table_name = 'backup_table_1');"
-                )
-                assert cursor.fetchone()[
-                    0
-                ], f"replication isn't working correctly: table 'backup_table_1' doesn't exist in {new_unit_name}"
-                cursor.execute(
-                    "SELECT EXISTS (SELECT FROM information_schema.tables"
-                    " WHERE table_schema = 'public' AND table_name = 'backup_table_2');"
-                )
-                assert not cursor.fetchone()[
-                    0
-                ], f"replication isn't working correctly: table 'backup_table_2' exists in {new_unit_name}"
-            connection.close()
-
-            switchover(ops_test, primary, new_unit_name)
-
-            # Get the new primary unit.
-            primary = await get_primary(ops_test, new_unit_name)
-            # Check that the primary changed.
-            for attempt in Retrying(
-                stop=stop_after_attempt(3), wait=wait_exponential(multiplier=1, min=2, max=30)
-            ):
-                with attempt:
-                    assert primary == new_unit_name
-
-            # Ensure stanza is working correctly.
-            logger.info("listing the available backups")
-            action = await ops_test.model.units.get(new_unit_name).run_action("list-backups")
-            await action.wait()
-            backups = action.results.get("backups")
-            assert backups, "backups not outputted"
-            await ops_test.model.wait_for_idle(status="active", timeout=1000)
+    # Remove the database app.
+    await ops_test.model.remove_application(database_app_name, block_until_done=True)
 
-        # Remove the database app.
-        await ops_test.model.remove_application(database_app_name, block_until_done=True)
+    # Remove the TLS operator.
+    await ops_test.model.remove_application(tls_certificates_app_name, block_until_done=True)
+
+
+@pytest.mark.group(2)
```

Comment:
nitpick: IIRC there's support for string groups, so we could use the cloud names  instead of 1-2

Reply:
really? Never seen it before. Will Try it out in another follow-up PR (for self_healing test). Thanks for the tip.